docs/simulators/OpenFOAM.md

```diff
@@ -126,6 +126,286 @@ task.download_outputs()
+like mesh quality reports, final results, and log files for each step in the process.
+
+To inspect the output, look under the `inductiva_output` folder where the
+results are downloaded:
+
+```bash
+ls -lasgo inductiva_output/<task_id>
+total 672
+  0 drwxr-xr-x  143     4576 Sep 19 16:06 .
+  0 drwxr-xr-x  400    12800 Sep 19 16:05 ..
+  0 drwxr-xr-x   11      352 Sep 19 16:06 0.orig
+  8 -rw-r--r--    1      626 Sep 19 16:06 Allclean
+ 16 -rw-r--r--    1     6998 Sep 19 16:06 Allrun
+  8 -rw-r--r--    1      991 Sep 19 16:06 COPYING
+ 48 -rw-r--r--    1    21547 Sep 19 16:06 README.md
+  0 -rw-r--r--    1        0 Sep 19 16:06 case.foam
+  0 drwxr-xr-x    8      256 Sep 19 16:06 constant
+  0 drwxr-xr-x   27      864 Sep 19 16:06 dynamicCode
+  0 drwxr-xr-x   13      416 Sep 19 16:06 figures
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor0
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor1
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor10
+    ...
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor126
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor127
+ 32 -rw-r--r--    1    14437 Sep 19 16:06 stderr.txt
+536 -rw-r--r--    1   272331 Sep 19 16:06 stdout.txt
+  0 drwxr-xr-x   30      960 Sep 19 16:06 system
+ 24 -rw-r--r--    1    11399 Sep 19 16:06 thumbnail.png
+```
+
+You can now perform post-processing on the results, just as you would if you had
+run the simulation locally.
+
+### Utilizing an MPI Cluster for Enhanced Simulation Performance
+
+The initial simulation conducted on a single `c3d-highcpu-360` machine took 40
+hours and 20 minutes to complete. To expedite this process, we opted to deploy
+the simulation across two `c3d-highcpu-360` machines in an MPI cluster configuration.
+This strategic change reduced the simulation time significantly to just 19 hours
+and 10 minutes.
+
+Leveraging Inductiva's Python library, scaling the simulation resources is
+straightforward. You simply need to modify the instantiation of the machine group,
+replacing it with an MPI cluster setup, and adjust the `n_vcpus` parameter accordingly.
+Below is an example of the code implementation:
+
+```python notest
+auto_terminate = (datetime.datetime.now() + datetime.timedelta(days=6)).replace(tzinfo=pytz.utc)
+mpi_cluster = inductiva.resources.MPICluster(
+    machine_type="c3d-highcpu-360",
+    data_disk_gb=300,
+    num_machines=2,
+    auto_terminate_ts=auto_terminate
+)
+
+mpi_cluster.start()
+
+...
+# Execute the simulation
+task = openfoam.run(
+    input_dir=input_dir,
+    commands=commands,
+    n_vcpus=360,
+    use_hwthread=True,
+    on=mpi_cluster
+)
+```
+
+### Conclusion
+
+This setup demonstrates how to leverage cloud resources to run computationally
+heavy OpenFOAM simulations, specifically using a case from the ExaFOAM benchmarks.
+The cloud-based environment enables parallel execution on high-performance
+machines, drastically reducing computation time. Once the simulation completes,
```

Comment:
You can be more specific. It's not just drastically reducing. It's halving. In fact is actually even better.